src/components/RecordingDialog.tsx

```diff
@@ -108,8 +110,11 @@ const RecordingDialog = ({
             onDone(data) {
               const recordingId = Date.now();
               addGestureRecordings(gestureId, [{ ID: recordingId, data }]);
-              handleCleanup();
-              onRecordingComplete(recordingId);
+              setRecordingStatus(RecordingStatus.Done);
+              doneTimeout.current = setTimeout(() => {
+                handleCleanup();
+                onRecordingComplete(recordingId);
+              }, 1500);
```

Comment:
I anticipate that some folks are going to find this quite painful as an extra delay to repeated recording.

Reply:
Hmmm... yes, not sure what to do about that 🤔 Reduce time, or maybe make it into a full on dialog and add a "do not show again"? Or maybe the new animation for highlighting a newly added recording is sufficient?

Reply:
I'll experiment with shorter times (+ copy?) when I get a mo.